scripts/check-for-emails.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Finds emails without example or mailinator in there
+# Example (from root of repo):
+#    ! ./check-for-emails.sh
+#
+# NOTE: return value needs to be negated
+
+EMAIL="\"[A-Z0-9a-z._%+-]+@[[A-Z0-9a-z._%+-]+\""
+grep --exclude-dir=node_modules --exclude-dir=_build -R \
+	-E "${EMAIL}" . | \
+	grep -v -E "example|mailinator"
```

Comment:
This should be used and run on precommit. Raise a seperate issue for it.

Reply:
See #1082